dotcom-rendering/.snyk

```diff
@@ -4,12 +4,4 @@ version: v1.1229.0
 # or ignored in the Snyk console.
 # ignores vulnerabilities until expiry date; change duration by modifying expiry date
 ignore:
-  'snyk:lic:npm:pm2:AGPL-3.0':
```

Comment:
does this file do anything now?

Reply:
It doesn't at the moment but I kept in case we need to need to configure snyk license ignores in future. I'll ask the team if they think its worth keeping around and if not I'll raise a separate PR to remove.